src/connector.ts

```diff
@@ -789,7 +793,12 @@ export default {
             );
         standaloneConnect(context, connectSequence);
     },
-    disconnect: (options = null, callback = () => {}) => {
+    disconnect: (
+        options = null,
+        callback = () => {
+            // do nothing
+        }
```

Comment:
Maybe we should have a no-op function somewhere that we reference everywhere we do this?

Reply:
I'm still wondering why we do it. 😄